client/layout/guided-tours/tours/checklist-site-icon-tour.js

```diff
@@ -82,8 +89,8 @@ export const ChecklistSiteIconTour = makeTour(
 			onTargetDisappear={ handleTargetDisappear }
 		>
 			<Continue target="dialog-base-action-confirm" step="click-done" click>
-				{ translate( 'Good choice, press {{b}}Continue{{/b}} to use it as your Site Icon.', {
-					components: { b: <strong /> },
+				{ translate( 'Good choice, press {{continueButton/}} to use it as your Site Icon.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Good choice, press {{ContinueButton/}} to use it as your Site Icon.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).